SQLQuery1.sql

```diff
@@ -0,0 +1,124 @@
+﻿USE [master]
```

Comment:
delete SQLQuery1.sql off your machine and do a another add and commit, this query exists in the repo already and we dont want to commit a duplicate, i will delete the file from this commit but make sure you remove it before you do another commit and pr or it will just get pushed up again since it has a different name than the original seed query

Reply:
It has been done.